.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,14 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Opencast Studio
+    url: https://github.com/elan-ev/opencast-studio/issues
+    about: Report issues with Opencast Studio in its own repository.
+  - name: Opencast Editor
+    url: https://github.com/opencast/opencast-editor/issues
+    about: Report issues with Opencast Editor in its own repository.
+  - name: Security issue report
+    url: https://github.com/opencast/opencast/blob/develop/SECURITY.md
+    about: Please report security vulnerabilities here.
+  - name: Community Support
```

Comment:
Maybe is better to have this part as the first on the list. Lot of the problems that face new users are normally configuration issues than bugs. What do you think?

Reply:
We cannot push it to the top. The external links are always below the issue templates. That would mean we would end up with something like:

- report bug
- report bug
- mailing list
- report bug
- report bug

That's a bit weird.

But I also don't see this as a big problem because right now, we don't really see the issue tracker being used for configuration questions. Just adding the templates shouldn't change that. So, most people coming do want to report a bug and for them the other options make more sense and should therefor be at the top.

Reply:
In that case, yes, is it better to left it where it is.